Added max number of Azure Automation Accounts

blueelvis opened the pull request:
Added the maximum number of Azure automation accounts that are possible in a subscription. Asked and got answer over here - https://social.msdn.microsoft.com/Forums/en-US/5635d3af-d336-46b9-8760-1b81a301b507/what-is-the-maximum-number-of-azure-automation-accounts-one-can-create-in-an-azure-subscription?forum=azureautomation

deneha commented:
Hi @kenazk - Thanks for the contribution. The links you adjusted need the .md after them.  This is how they should look if the file is in the same directory.  ''[link text](article-name.md)

kenazk commented:
Thanks Dene. Updated the link.

blueelvis committed:
Added max number of Azure Automation Accounts

Added the maximum number of Azure automation accounts that are possible in a subscription. Asked and got answer over here - https://social.msdn.microsoft.com/Forums/en-US/5635d3af-d336-46b9-8760-1b81a301b507/what-is-the-maximum-number-of-azure-automation-accounts-one-can-create-in-an-azure-subscription?forum=azureautomation
